packages/backend/src/nest/tor/tor-control.service.ts

```diff
@@ -104,13 +104,16 @@ export class TorControl {
   }
 
   public async sendCommand(command: string): Promise<{ code: number; messages: string[] }> {
+    this.logger(`Sending tor command: ${command}`)
     // Only send one command at a time.
     if (this.isSending) {
       this.logger('Tor connection already established, waiting...')
     }
 
     // Wait for existing command to finish.
     while (this.isSending) {
+      const timeout = 750
+      this.logger(`Waiting for ${timeout}ms to retry command...`)
       await new Promise(r => setTimeout(r, 750))
```

Comment:
Should we use the `timeout` variable here?

Reply:
Whoops, that was the intent.